Carl-bot/dist/metadata.json

```diff
@@ -1,22 +1,22 @@
 {
-    "author": {
-      "name": "NMW03",
-      "id": "534099893979971584"
-    },
-    "service": "Carl-bot",
-    "description": {
-      "en": "A fully customizable and modular discord bot featuring reaction roles, automod, logging, custom commands and much more."
-    },
-    "url": [
-      "carl.gg"
-    ],
-    "version": "1.2",
-    "logo": "https://i.imgur.com/2yPwE4a.png",
-    "thumbnail": "https://i.imgur.com/wHYCN0D.png",
-    "color": "#757F3F",
-    "tags": [
-      "bot",
-      "discord"
-    ],
-    "category": "other"
-  }
+  "author": {
+    "name": "NMW03",
+    "id": "534099893979971584"
+  },
+  "service": "Carl-bot",
+  "description": {
+    "en": "A fully customizable and modular discord bot featuring reaction roles, automod, logging, custom commands and much more."
+  },
+  "url": [
+    "carl.gg"

```

Comment:
Make the url a string and not an array.

Reply:
oh, I'm was try to find where the error is